app/views/layouts/snowcrash.html.erb

```diff
@@ -19,7 +19,12 @@
 
   <%= csrf_meta_tag %>
 </head>
-<body class="<%= controller_name %> <%= action_name %>">
+<body
+  class="<%= controller_name %> <%= action_name %>"
+  <% if current_user %>
+    data-current-user-id="<%= current_user.id %>"
```

Comment:
What about using:

https://github.com/secroots/dradis-pro/blob/develop/app/views/activities/_poller.html.erb

Reply:
you mean put the `data` attribute there instead? Probably makes more sense.